src/pipeline.ts

```diff
@@ -194,6 +194,19 @@ export class GitHubWorkflow extends PipelineBase {
     // eslint-disable-next-line no-console
     console.error(`writing ${this.workflowPath}`);
     mkdirSync(path.dirname(this.workflowPath), { recursive: true });
+
+    if (process.env.SYNTH === 'github') {
```

Comment:
I'm 99% sure there are just environment variables that will already be set for you to indicate that you are running on GitHub Actions. Why not check those?